addons/spreadsheet/static/src/global_filters/helpers.js

```diff
@@ -71,49 +71,49 @@ export function checkFilterFieldMatching(fieldMatchings) {
  * @returns {Domain|undefined}
  */
 export function getRelativeDateDomain(now, offset, rangeType, fieldName, fieldType) {
```

Comment:
@yunchen4 @LucasLefevre The logic of this function was updated,  but the docstring still reflects the old implementation.